---
title: "Population structure analysis of _Metarhizium_"
author: "Rodrigo Ortega Polo"
date: "October 21, 2018"
output: word_document
bibliography: Metarhizium.bib
---

## Introduction

This analysis was performed to address reviewer #2's comments on
Inglis et al., 2018 ("Biogeography and subspecies diversity of 
_Metarhizium brunneum_ and _Metarhizium robertsii_ in northwestern 
North America") submitted to the Canadian Journal of Microbiology.

Reviewer #2 points out that _"allelic diversity statistics are not given and no standard
measures of population structure or recombination are presented. This would greatly strengthen the paper."_

Reviewer #2's further notes: "How about some allele diversity statistics, test of linkage (dis) equilibrium and
tests of genetic structure within species? How geographically dispersed are the clones? Given existing gaps in
understanding _Metarhizium_ population genetic structure, this omission is curious".

## Methods

### Preparing data for analysis

The analyses were performed with the data provided by Jenny Gusse. After investigation,
isolate #2027 was correctly assigned as _M. robertsii_. Some microsatellite locus entries 
were dropped from the dataset because either they were completely absent from the _M. robertsii_ 
dataset (Ma375_3, Ma307_2, and Ma2296_2) or that were present only in one isolate in the _M. brunnei_ 
dataset (Ma307_2 and Ma2296_2).

### Linkage Disequilibrium and Hardy-Weinberg Equilibrium

### Population genetic structure

In order to analyze the population genetic structure, the STRUCTURE v.2.3.4 program 
[@pritchard_inference_2000] was run with the admixture model, 
correlated allele frequencies and locations as priors (LOCPRIORS). This was done for 
the analyses of both _M. robertsii_ and _M. brunnei_. The data of each species was used for the comparisons 
of isolates from both agricultural vs. natural habitats and glaciated vs. non-glaciated locations. 
STRUCTURE is a Bayesian clustering method that finds the most likely number of population 
clusters (K) based on allelic frequency that maximizes Hardy-Weinberg equilibrium and minimizes linkage disequilibrium.
Five independent runs for each value of K from 1 to 5 were 
run (10,000 burn-in periods with 20,000 Markov Chain Monte Carlo (MCMC) repetitions after 
burn-in.). Fixation index (Fst) values were calculated for analyzing genetic differentiation. 
Results were averaged and the true K value was determined using STRUCTURE HARVESTER web version 
v0.6.94 [@earl_structure_2012] by using the delta K (Evanno) and the mean log likelihood LnPr(X|K).
Optimal alignments for the multiple replicate cluster analyses were searched with 
CLUMPP [@jakobsson_clumpp_2007] to ensure cluster labels match across runs, with LargeKGreedy method
with the G' pairwise matrix similarity statistic, random input orders and 100 repeats. 
Cluster and ancestry coefficient data processed with CLUMPP was plotted with the ggplot2 package 
version 3.0.0.9000 [@wickham_ggplot2_2016] of the R statistical computing language [@r_core_2018].

The Python and R code that was used for data preparation and visualization is available 
in: https://github.com/ropolomx/metarhizium. 

## Results and Discussion

### Glaciated vs. non-glaciated locations

The STRUCTURE and subsequent STRUCTURE HARVESTER analyses revealed that for 
_M.robertsii_ the true value of K was 3 clusters and for _M. brunneum_ K=2, when considering glaciated vs. 
non-glaciated locations as prior locations. In the case of _M. robertsii_, the isolates inside areas 
that were glaciated included an admixed individual (sample #347, Osoyoos, BC, 
Montane Cordilla South Ecozone, Agricultural). A closer analyses of the metadata associated with those samples
revealed that. In the case of _M. brunneum_.



### Agricultural vs. natural sites

The true value of K was 3 for _M. robertsii_ and 2 for _M. brunneum_ when using agricultural vs. 
natural sites as prior locations. 



In conclusion, the observations on population structure were 
consistent with the results described with the minimum spanning trees 
in the manuscript. Namely, there is no distinction between 
agricultural and natural isolates with minimum spanning tree, but 
there is clear distinction between isolates inside glaciation vs. 
outside glaciation.

## Acknowledgments

ROP is supported by AAFC startup funding for the Bioinformatics Research Support Unit 
of the Lethbridge Research and Development Centre.

## Bibliography


